azure-pipelines-e2e-appeals.yml

```diff
@@ -15,21 +15,23 @@ pr:
   branches:
     include:
       - main
+      - feat/e2e-cypress-refactoring
```

Comment:
We shouldn't need this here, this is which branch the PR is merging _into_

Reply:
Thanks, good catch